app/src/main/res/values-land/dimens.xml

```diff
@@ -377,4 +377,9 @@
   <dimen name="profile_reset_pin_activity_profile_reset_input_confirm_pin_margin_end">168dp</dimen>
   <dimen name="profile_reset_pin_activity_profile_reset_save_button_margin_end">168dp</dimen>
 
+  <!-- SectionTitleFragment -->
+  <dimen name="section_title_divider_view_layout_margin_top">28dp</dimen>
+  <dimen name="section_title_text_view_layout_margin_start">72dp</dimen>
+  <dimen name="section_title_text_view_layout_margin_top">28dp</dimen>
+  <dimen name="section_title_text_view_layout_margin_end">72dp</dimen>
```

Comment:
@JishnuGoyal add one empty line above </resources> as a part of formatting like it was before edits
Similar for the other dimens file as well

Reply:
done